### PR TITLE
Add direct worklog cron

### PR DESCRIPTION
**Pull request by PranavBakre**
## Summary
- Adds an env-gated daily worklog cron that collects configured repo commits and GitHub PRs.
- Writes a markdown worklog snapshot and posts a grouped Slack summary.
- Adds deterministic fallback summary formatting plus optional runner-based summarization.
- Documents worklog env vars and submodule layout guidance.

## Verification
- `bun run typecheck`
- `bun test src/cron src/worklog src/config.test.ts`
- `bun run build`
- `bun test`

## Notes
- Rebased onto `personal/main` after PR #45 was merged.
- The older local `docs/features/dynamic-workflows.md` draft was resolved in favor of the newer merged PR #45 doc.

### CODE REVIEW

**Review by PranavBakre — COMMENTED** [review]
Reviewed. Found two worklog issues worth fixing before merge.

File: src/worklog/activity.ts [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+  for (const key of ["user.email", "user.name"]) {
+    const result = await runCommand("git", ["config", "--get", key], cwd);
+    if (result.exitCode === 0 && result.stdout.trim()) {
+      return result.stdout.trim();
+    }
+  }
+  return null;
+}
+
+export function parseGithubSlug(remoteUrl: string): string | null {
```
Comment (PranavBakre) [inline src/worklog/activity.ts:260]: This parser drops valid GitHub repositories whose repo name contains a dot. For example `https://github.com/org/site.io.git` normalizes to no match because `[^/.]+` rejects `site.io`; `collectPullRequests` then returns `[]` with no error, so the daily worklog silently omits PRs for those repos. PR #45 already fixed the same edge case in the dynamic workflow worklog path by stripping a trailing `.git` first and then matching `owner/repo` segments; this direct cron path should use the same parser/test coverage.

File: src/worklog/summary.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+): Promise<string | null> {
+  const channel = config.worklog.channel ?? "worklog-cron";
+  const session = createSession(
+    `cron-worklog-${activity.until.slice(0, 10)}`,
+    channel,
+    "quiet",
+    config.runner.provider,
+    config.claude.defaultDriver,
+  );
+  session.cwd = process.cwd();
```
Comment (PranavBakre) [inline src/worklog/summary.ts:59]: The summarizer runner is launched from `process.cwd()`, which means this utility summarization runs with Junior’s project workspace/MCP context even though it only needs the JSON activity payload. That is the same footgun we just fixed in PR #45 by using a utility cwd (`/tmp/junior-utility`) so project MCP wiring is skipped for non-repo workflows. Please move this to a utility cwd; otherwise a Slack-summary prompt can accidentally inherit repo tools/context and broaden what should be a read-only compression step.

**Comment by PranavBakre** [comment]
Closing this direct cron PR. Worklog should continue through the workflow definition path instead.
